py/commands/update.py

```diff
@@ -0,0 +1,10 @@
+@bot.command()
+async def update(ctx):
+    data = ctx.guild.get_role(725512500220657674)
+    if ctx.author.roles in data:
```

Comment:
I would hazard a guess and say this doesn't do what you expect it to. `Guild.get_role` will return an `Optional[Role]`, so you probably want to check that `data is not None` and maybe check that `data in ctx.author.roles` rather than the other way around; currently, you're trying to see if a `List[Role]` is inside of an `Optional[Role]`, when it should be checking if a `Role` is in a `List[Role]`.